core/iwasm/compilation/aot_emit_const.c

```diff
@@ -12,6 +12,7 @@ aot_compile_op_i32_const(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
 {
     LLVMValueRef value;
 
+#if !defined(BUILD_TARGET_XTENSA)
```

Comment:
How about implement this by modify the intrinsic flags instead?
https://github.com/bytecodealliance/wasm-micro-runtime/blob/8acaaa2892758eb309759a4ae760f2b9e25b9785/core/iwasm/aot/aot_intrinsic.c#L870-L884